cmd/kube-proxy/app/server.go

```diff
@@ -59,8 +56,6 @@ func NewProxyServer() *ProxyServer {
 
 // AddFlags adds flags for a specific ProxyServer to the specified FlagSet
 func (s *ProxyServer) AddFlags(fs *pflag.FlagSet) {
-	fs.StringVar(&s.EtcdConfigFile, "etcd_config", s.EtcdConfigFile, "The config file for the etcd client. Mutually exclusive with -etcd_servers")
```

Comment:
In hindsight we should have left the flags as no-ops with a warning.  This is an annoyance in that old configs no longer work correctly.